themes/bootstrap3/js/combined-search.js

```diff
@@ -3,11 +3,15 @@ VuFind.combinedSearch = (function CombinedSearch() {
 
   function init(container, url) {
     VuFind.loadHtml(container, url, '', function containerLoad(responseText) {
-      if (responseText.length === 0) {
-        container.style.display = "none";
+      if (!responseText || responseText.length === 0) {
+        if (container.style) {
```

Comment:
Is it really possible for .style to be null/undefined on an element?

Reply:
Fair enough!  I wonder then if we should be **creating** the style element if it's not present but who knows if that would even work or what side effects it would have.  I guess we can cross that bridge, and this is certainly an incremental improvement at leats.

Reply:
How about using `container.classList.add('hidden');` instead of changing style?

Reply:
See #4147 for the real solution to this problem. I'll leave it up to @maccabeelevine whether using the classList instead of the style is preferable, but I think that would be a separate change unrelated to my fix.